pipelines/LATISS/ApPipe.yaml

```diff
@@ -0,0 +1,15 @@
+description: AP pipeline specialized for LATISS
+instrument: lsst.obs.lsst.Latiss
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipe.yaml
+    include:
+      - isr
+      - characterizeImage
+      - calibrate
+      - retrieveTemplate
+      - subtractImages
+      - detectAndMeasure
+      - transformDiaSrcCat
+      - diaPipe
+parameters:
+  coaddName: deep
```

Comment:
This seems an odd thing to put in the main pipeline, since the correct value of `coaddName` is a function of the data the pipeline is run on. Is it expected that all LATISS coadds will be `deep`? If not, I recommend this be moved to lsst-dm/prompt_prototype#73.